### ISSUE TITLE
New AlaSQL documentation repository

### ISSUE THREAD

**Issue by agershun** [AlaSQL/alasql#264]
@mathiasrw I opened new repository for AlaSQL future quality documentation. Due AlaSQL is similar to PostgreSQL, we can use parts of its great open-source documentation to adapt to AlaSQL syntax. 

https://github.com/agershun/alasql-manual/wiki.

I understand this is plagiarism, but this is permitted by the license. All references to the original project, of course, will be included.

Please, tell me if this is not good approach

Hope it can save a lot of time and help us to provide AlaSQL users with better documentation.

**Comment by mathiasrw** [comment]
Its great!!

How eco e you made new repo? We could use the wiki of this repo - or If your idea is that it must be us storhed in files we could make a manual branch...

**Comment by agershun** [comment]
1. I already realized that it is a killing idea to use diffrerent branch for gh-pages. IN combination with Dropbox it is very explosive.
2. I wanted to test this idea first. PG Manual has more than 100 pages, and it will take time to adapt them consciously to AlaSQL
3. I worried about pages names to do not interfere current pages in our wiki

I think we can create about 10-20 pages, understand all possible problems, and then merge it into current Wiki.
